### PR TITLE
[UI Improvement] Auto resizing event and volunteer list

### LINKED ISSUE #131: UI Does Not DIsplay Full Information
When scrolling down the volunteers list, not all information is shown.
See image below for example
image

### PR DESCRIPTION
**Pull request by Fluffykan**
Previously the height and width of the event and volunteer lists remained constant even when the window resizes. This commit enables the lists to grow / shrink according to the changing window size.

Just a nice to have feature, does not affect functionality.

closes #131 by increasing minHeight to 700px in `MainWindow.fxml`

### CODE REVIEW

**Review by ttzuche — COMMENTED** [review]
Is the print statement on the VolunteerParserUtil needed?

File: src/main/java/seedu/address/logic/parser/VolunteerParserUtil.java
Comment (JovenSoh) [inline src/main/java/seedu/address/logic/parser/VolunteerParserUtil.java:1]: Would it be possible to take the print statement out?

**Review by JovenSoh — CHANGES_REQUESTED** [review]

**Comment by Fluffykan** [comment]
@tzuennn @JovenSoh my bad i left that in for debugging, will remove it.

**Review by JovenSoh — APPROVED** [review]
LGTM!

**Pull request merged**